vignettes/geocoding_simple_output.Rmd

```diff
@@ -0,0 +1,350 @@
+---
+title: 'Geocoding: Voter Addresses'
+author: "null"
+date: "8/4/2020"
+output:
+  pdf_document: default
+  html_document: default
+---
+
+In this vignette, we will walk-through how to geocode a dataset that includes addresses in order to estimate race using BISG.
+
+## What is Geocoding?
+
+One of the first steps to performing ecological inference using eiCompare is geocoding your voter file addresses in order to perform Bayesian Improved Surname Geocoding (BISG).  Geocoding is the process of using an address or place of location to find geocographic coordinates (i.e. latitude/longitude) of that location on a map. In relation to performing BISG, the values of the geographic coordinates are compared to other census data containing self-reported race and ethnicity to determine the likelihood of an individual living in an ecological unit area (i.e. county, block, tract) being of a certain race given their address. This probability is then used to update a prior probability in the BISG analysis. For more information on BISG, please refer to the BISG vignette. 
+
+Below are some steps to help you walk through the process of performing geocoding on your voter file. 
+
+### Step 1: Load R libraries/packages
+Each library/package loaded allows you to use certain functions needed to prep your data for geocoding and run the geocoding tool(s).
+```{r}
+# Load libraries
+suppressMessages(c(
+  library(stringr),
+  library(readr),
+  library(tidyverse),
+  library(foreach),
+  library(parallel),
+  library(doParallel),
+  library(data.table),
+  library(gmodels),
+  library(plyr),
+  library(censusxy),
+  library(tidycensus),
+  library(sf)
+))
+```
+
+### Step 2: Load your voter data.
+We are using the Georgia and Fulton county voter registration dataset and geocoding all voter addresses.
+
+```{r}
+# Load full dataset for all registered voters in Georgia
+path <- "~/shared/BISG_datasets/"
```

Comment:
same issue as other vignette; will need to figure out how to get this dataset into the package.